client/src/features/projectsV2/fields/SlugFormField.tsx

```diff
@@ -17,44 +17,76 @@
  */
 
 import cx from "classnames";
-
-import { Controller } from "react-hook-form";
+import { ArrowCounterclockwise } from "react-bootstrap-icons";
 import type { FieldValues } from "react-hook-form";
+import { Controller } from "react-hook-form";
+import { Button, FormText, Input, InputGroup, Label } from "reactstrap";
 
-import { FormText, Input, Label } from "reactstrap";
-import type { GenericFormFieldProps } from "./formField.types";
+import type { SlugFormFieldProps } from "./formField.types";
 
 export default function SlugFormField<T extends FieldValues>({
+  compact,
   control,
   entityName,
   errors,
+  resetFunction,
   name,
-}: GenericFormFieldProps<T>) {
+  url,
+}: SlugFormFieldProps<T>) {
+  const content = (
+    <Controller
+      control={control}
+      name={name}
+      render={({ field, fieldState: { isDirty } }) => {
+        return (
+          <InputGroup>
+            <div className="input-group-text">
+              <small>{url}</small>
+            </div>
+            <Input
+              aria-describedby={`${entityName}SlugHelp`}
+              className={cx(
+                "form-control",
+                errors.slug && isDirty && "is-invalid",
```

Comment:
This prevents showing an error when the slug is empty

```suggestion
                errors.slug && "is-invalid",
```